scripts/innthomas.py

```diff
@@ -4,5 +4,5 @@
 langage="Python"
 email="innthomas00158@gmail.com"
 def myData():
-    print ("Hello Word, this is {} with HNGi7 ID {} using {} for stage 2 task".format(fullName, ID, langage))
+    print ("Hello Word, this is {} with HNGi7 ID {}  and email {} using {} for stage 2 task".format(fullName, ID,email ,langage))
```

Comment:
1. Please fix the typo "Word" to "World"
2. There's one too many spaces between ID curly braces and the phrase "and email"